stories/Input/SelectStory.tsx

```diff
@@ -27,9 +27,28 @@ const props = {
       name: 'status',
       type: 'string',
       defaultValue: '',
-      possibleValue: <code>success | error</code>,
+      possibleValue: <code>error</code>,
       require: 'no',
-      description: 'Sets different style for Select based on status.',
+      description:
+        '(DEPRECATED) Sets different style for Select based on status.',
+    },
+    {
+      name: 'error',
+      type: 'ReactNode | boolean | string',
+      defaultValue: '',
+      possibleValue: 'any',
+      require: 'no',
+      description:
+        'Sets error state on component. If string, the error will be shown below the select',
+    },
+    {
+      name: 'renderError',
+      type: 'ReactNode',
```

Comment:
I think this should be ``type: function` instead?

Reply:
I was following this: https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#useful-react-prop-type-examples Do you think people won't understand `ReactNode`?

Reply:
Ahh I see. I didn't understand at first but after reading it makes sense :+1: